RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Visibility of the “incoming chats” in queue

## Proposed changes


## Issue(s)	


## How to test or reproduce


- Be a `livechat-agent`
- Enable/Disable your availability to omnichannel -> press on the Row or just on the Switch, should happen the same action
- If you are activating, should appear a new item in the list that is the row where you can see the number of users in the queue. If there isn't any user, the row cannot be pressed and the right icons must be changed to `Empty`

## Screenshots

#### After for iOS
https://user-images.githubusercontent.com/47038980/162543980-24f2df17-8e12-469a-9b70-2403cee47c4f.mov

#### After for Android
https://user-images.githubusercontent.com/47038980/162544672-3c859da1-9a6e-4c24-9acb-5b475d88ff27.mov


## Types of changes




## Checklist



## Further comments
